examples/charts/brushing_zooming.htm

```diff
@@ -0,0 +1,166 @@
+            <div class='row trunk-section'>
+                <div class='col-lg-7'>
+                    <div class='row'>
+                        <div class='col-lg-12 text-center' id='basic_brushing'></div>
+                        <div class='col-lg-12 text-center' id='point_chart_brushing'></div>
+                    </div>
+                </div>
+                <div class='col-lg-5'>
+                    <div class='data-column'><a href='data/fake_users3.json'>data</a></div>
+
+<pre><code class='javascript'>d3.json('data/fake_users2.json', function(data) {
+    for (var i = 0; i < data.length; i++) {
+        data[i] = MG.convert.date(data[i], 'date');
+    }
+    MG.data_graphic({
+        title: "Basic Brushing & Zooming",
+        description: "This is a simple for brushing and zooming feature. You can set 'brush' as 'xy', 'x', 'y' to restrict axis.",
+        data: data,
+        top: 70,
+        width: 600,
+        height: 240,
+        right: 40,
+        missing_is_hidden: true,
+        target: '#basic_brushing',
+        brush: 'xy',
+    });
+});
+
+d3.json('data/points1.json', function(data) {
+    MG.data_graphic({
+        title: "Point Chart Brushing",
+        description: "Unlike addons, this feature can also be used for point charts.",
+        data: data,
+        chart_type: 'point',
+        width: 600,
+        height: 240,
+        right: 40,
+        target: '#point_chart_brushing',
+        x_accessor: 'x',
+        y_accessor: 'y',
+        y_rug: true,
+        brush: 'xy',
+    });
+});</code></pre>
+
+                </div>
+            </div>
+
+            <div class='row trunk-section'>
+                    <div class='col-lg-7'>
+                        <div class='row'>
+                            <div class='col-lg-12 text-center' id='main'></div>
+                            <div class='col-lg-12 text-center' id='overview_plot'></div>
+                        </div>
+                    </div>
+                    <div class='col-lg-5'>
+                        <div class='data-column'><a href='data/fake_users3.json'>data</a></div>
+
+<pre><code class='javascript'>d3.json('data/fake_users2.json', function(data) {
+    for (var i = 0; i < data.length; i++) {
+        data[i] = MG.convert.date(data[i], 'date');
+    }
+    const main = {
+        title: "Overview Plot",
+        description: "This is a simple for creating an overview plot. You can create an overview plot easily by creating another chart with 'zoom_target' option and set it as the object of main chart. Besides, you can hide axis and active point to simplify the overview plot.",
```

Comment:
"This is a simple for creating an overview plot. You can create an overview plot easily by creating another chart with 'zoom_target' option and set it as the object of main chart. Besides, you can hide axis and active point to simplify the overview plot."  

=>

"This is a simple example of an overview plot. You can create an overview plot by creating another chart with 'zoom_target' option and then setting it as the object of the main chart."

Reply:
P.S. I didn't realize you left the overview plot functionality in. :) That's ok-- there are a few features I would like to add to that in a followup. We can discuss next week. :)